### PR TITLE
Limit upload image

### LINKED ISSUE #6: Update user profile picture
- [x] Users should be able to see their current profile picture at the page. If they didn't upload the picture yet, then they should see a placeholder.
- [x] Users should be able to update their profile picture.
- [x] Users should not be able to upload images which size is more than 1024x1024px

### PR DESCRIPTION
**Pull request by Washubrnfg85**

### CODE REVIEW

File: src/main/resources/templates/profile_details_page.html [hunk 1]
```diff
@@ -137,6 +141,11 @@ <h4 class="alert-heading">Error while loading profile details</h4>
                 state.publicEmail = publicEmail;
             }
 
+            function getMessageFromResponse(resp) {
```
Comment (eugenesmkn) [inline src/main/resources/templates/profile_details_page.html]: Where do you use this?

File: src/main/resources/templates/profile_details_page.html [hunk 2]
```diff
@@ -120,6 +123,7 @@ <h4 class="alert-heading">Error while loading profile details</h4>
     const app = createApp({
         setup() {
             const state = reactive({
+                message: null,
```
Comment (eugenesmkn) [inline src/main/resources/templates/profile_details_page.html]: The name is too generic

File: src/main/resources/templates/profile_details_page.html [hunk 3]
```diff
@@ -75,6 +75,9 @@ <h1 class="text-center">Profile Details</h1>
                 <label class="form-label">Profile Picture</label>
                 <input class="form-control" type="file" @change="handleFileUpload" :disabled="state.uploadingImage">
                 <p class="form-text">Image must be below 1024x1024px. Use PNG or JPG format.</p>
```
Comment (eugenesmkn) [inline src/main/resources/templates/profile_details_page.html]: There is also file format validation needed. I suggest removing support of PNG and do only JPG.

**Review by eugenesimakin — CHANGES_REQUESTED** [review]

File: src/main/resources/templates/profile_details_page.html (hunk 1 above)
Comment (Washubrnfg85) [reply src/main/resources/templates/profile_details_page.html]: Forgot about it. Removed.

File: src/main/resources/templates/profile_details_page.html (hunk 2 above)
Comment (Washubrnfg85) [reply src/main/resources/templates/profile_details_page.html]: Changed for imageUploadResponse

File: src/main/resources/templates/profile_details_page.html (hunk 3 above)
Comment (Washubrnfg85) [reply src/main/resources/templates/profile_details_page.html]: Yep, on way to go.

**Comment by Washubrnfg85** [comment]
Should be better now.

**Review by eugenesimakin — APPROVED** [review]

**Pull request merged**